### ISSUE TITLE
pushToStitch.sh runs, but cannot finish app creation properly

### ISSUE THREAD

**Issue by mfazza** [MaBeuLux88/mongodb-stitch-examples#2]
Console Log:

you have successfully logged in as XXXXXXXXXXXX
Unable to find app with ID: "moviecollection-idmpd": would you like to create a new app? [y/n]: y
App name [MovieCollection]: 
Available Projects:
MongoDBWorkshop - 5c64d56479358e3fb03bf4d4
Atlas Project Name or ID [MongoDBWorkshop]: 
Location [IE]: 
Deployment Model [GLOBAL]: 
New app created: moviecollection-muscx
Importing app...
failed to import app: error: cannot update field "app_default_domain"

I tried disabling hosting in the script, but it still doesn't work.  Isn't "app_default_domain" is automatically assigned by Stitch?

Let me know if you have any idea how to solve it, I'd love to check your demo out!

**Comment by MaBeuLux88** [comment]
I reproduced the issue @mattfazza. This was not happening before. I am looking into it. Thanks for raising this up!

**Comment by MaBeuLux88** [comment]
The MongoDB Stitch team is working on resolving that problem. It's actually very simple. 

If you look at MovieCollection/stitch.json, it contained some information (app_default_domain) from my Stitch project but it should not because I exported this as a template (see importFromStitch.sh).

So as a temporary solution, I removed this app_default_domain manually from the MovieCollection/stitch.json and now it works.

See https://github.com/MaBeuLux88/mongodb-stitch-examples/commit/90eadbd30370be0218b59547b42a5fcf06c0bc10. The other modification won't change anything, it's just some minor stuff they added in the project export format.

Let me know how it goes so I can close the ticket and thanks for letting me know :-) ! :+1:
